cmd/kind/create/cluster/createcluster.go

```diff
@@ -56,7 +56,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
 	cmd.Flags().StringVar(&flags.ImageName, "image", "", "node docker image to use for booting the cluster")
 	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "retain nodes for debugging when cluster creation fails")
-	cmd.Flags().BoolVar(&flags.Use, "use", false, "set KUBECONFIG to use newly created cluster")
+	cmd.Flags().BoolVar(&flags.Use, "use", false, "use newly created cluster after creation")
```

Comment:
i think the part about `KUBECONFIG` makes this clearer.
i would reword in the lines of:

> set KUBECONFIG to the configuration of this cluster, once it's created